docs/reference/api_references.rst

```diff
@@ -74,8 +74,9 @@ add a text plugin with the content ``'hello world'``. This is done in English::
 
     from cms.api import create_page, add_plugin
 
-    page = create_page('My Page', 'my_template.html', 'en')
-    placeholder = page.placeholders.get(slot='body')
+    page = create_page('My Page', 'my_template.html', 'en', created_by=user)
```

Comment:
Can you add a comment that `user` needs to be a user object (as opposed to a string, which was also valid in django CMS 3)?

Reply:
Apologies for the delay. The requested changes have been implemented. Thank you for your guidance!! :)